apps/dev-api/domains/miniapp/schema/PublishB2CAppService.js

```diff
@@ -320,7 +321,11 @@ const PublishB2CAppService = new GQLCustomSchema('PublishB2CAppService', {
             resolver: async (parent, args, context) => {
                 const { data: { app: { id }, options, environment } } = args
 
-                const app = await B2CApp.getOne(context, { id, deletedAt: null })
+                const app = await B2CApp.getOne(
+                    context,
+                    { id, deletedAt: null },
+                    `id ${EXPORT_FIELDS} name developer createdBy { name } logo { publicUrl originalFilename }`
```

Comment:
mb use `getExportIdField(environment)` or specify these fields explicitly?

```suggestion
const exportIdField = getExportIdField(environment)
// ...
                    `id ${exportIdField} name developer createdBy { name } logo { publicUrl originalFilename }`
```

Reply:
I would indicate it explicitly to make it easier to read